19_pascal/solution/pascal-solution.js

```diff
@@ -0,0 +1,31 @@
+const pascal = function (counter, currentLine = [1]) {
+  if (counter === 1) return currentLine;
+
+  const halfOfNextLine = currentLine.reduce(
+    (accumulator, currentElement, index, originalArray) => {
+      const nextElement = originalArray[index + 1]
+      if (currentElement <= nextElement) {
+        return [...accumulator, currentElement + nextElement];
+      }
+      return accumulator;
+    },
+    [1],
+  );
+
+  // Notice that pascal triangle's lines are always palindromic in nature. 
+  // We only need the first half to obtain the information to 
+  // Construct the second half
```

Comment:
Nit - based on other lines, doesn't need to be split, and is easier to read unsplit

```suggestion
  // We only need the first half to obtain the information to construct the second half
```